Marlin/Marlin_main.cpp

```diff
@@ -13583,8 +13583,14 @@ void setup() {
       pe_deactivate_magnet(1);
     #endif
   #endif
-}
-
+#if defined (MKS_12864OLED)
+  SET_OUTPUT(LCD_PINS_DC);   
+  SET_OUTPUT(LCD_PINS_RST);  
+  WRITE(LCD_PINS_RST, LOW);
+  delay(1000);
+  WRITE(LCD_PINS_RST, HIGH);
```

Comment:
Is this supposed to be a different pin from `LCD_PINS_RS`?

Reply:
patch file from MKS had it,

Reply:
Can you point me to the author of that patch? I can redirect some of these questions to them.

Reply:
I commented out that whole reset section and renamed the pin to LCD_PINS_RS in pins_ramps.h, and it does not seem to have affected the display, it still works.  The code was copied from the OEM recommendation, probably for an earlier marlin release.

Reply:
It's an interesting piece. If it does no harm, I guess it can be left in. Anyway, off to figure out what's wrong with MKS_12864OLED.